features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1832,7 +1832,8 @@
         </Resource>
         <Resource context="(.*)/oauth2/introspect(.*)"
                 secured="{{resource_access_control.introspect.secured}}"
-                http-method="all">
+                http-method="all"
+                allowed-auth-handlers="BasicClientAuthenticationHandler">
```

Comment:
value should be configurable from deployment.toml. We need to read the default values from default.json

Reply:
This should be similar to how the value of 'secured' element is read.
secured="{{resource_access_control.introspect.secured}}